rust/src/delta_arrow.rs

```diff
@@ -47,8 +49,12 @@ impl From<&schema::SchemaDataType> for ArrowDataType {
                     "byte" => ArrowDataType::Int8,
                     "float" => ArrowDataType::Float32,
                     "double" => ArrowDataType::Float64,
-                    "boolean" => ArrowDataType::Boolean,
```

Comment:
Why does your change remove "boolean" and "binary"?

Reply:
Oh, good catch! We don't want to change this :) I fix it.